### PR TITLE
Automate the generation of _GTLD_INCLUDE_SUBDOMAINS value

### PR DESCRIPTION
**Pull request by dalf**
Implement #11

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -70,6 +74,8 @@ def main():
                         name if is_leaf else label,
                     )
                 )
+                if i == 0 and is_leaf:
```
Comment (sethmlarson) [inline build.py:77]: We also need to check that `include_subdomains` is truthy here as well.

Comment (dalf) [reply build.py]: I forgot this check. Fixed in 2c7abdc

File: build.py [hunk 2]
```diff
@@ -122,6 +130,10 @@ def main():
     )
     data = CHECKSUM_RE.sub('__checksum__ = "%s"' % content_checksum, data)
     data = JUMPTABLE_RE.sub("_JUMPTABLE = %s  # noqa: E501" % str(jump_table), data)
+    data = GTLD_INCLUDE_SUBDOMAINS_RE.sub(
```
Comment (sethmlarson) [inline build.py:133]: Last nit, could this be updated to render the labels in sorted order (even if they are in a set)?
That way the order in the source file doesn't depend on the order within the list.

Comment (dalf) [reply build.py:137]: Thank you for the Blacken commit, I've checked ````nox -s test``` but I forgot to call ```nox -s check``` before.

**Pull request merged**